broker-cli/commands/wallet.js

```diff
@@ -261,62 +293,49 @@ async function networkStatus (args, opts, logger) {
 
     statusTable.push(['Pending', '', ''])
-    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.pendingReceiveCapacity, NETWORK_STATUSES.PENDING), formatBalance(counterSymbolCapacities.pendingSendCapacity, NETWORK_STATUSES.PENDING)])
-    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.pendingSendCapacity, NETWORK_STATUSES.PENDING), formatBalance(counterSymbolCapacities.pendingReceiveCapacity, NETWORK_STATUSES.PENDING)])
+    statusTable.push([`  Buy ${baseSymbol}`, formatBalance(baseSymbolCapacities.pendingReceiveCapacity, NETWORK_STATUSES.PENDING), formatBalance(counterSymbolCapacities.pendingSendCapacity, NETWORK_STATUSES.PENDING)])
+    statusTable.push([`  Sell ${baseSymbol}`, formatBalance(baseSymbolCapacities.pendingSendCapacity, NETWORK_STATUSES.PENDING), formatBalance(counterSymbolCapacities.pendingReceiveCapacity, NETWORK_STATUSES.PENDING)])
 
     statusTable.push(['Inactive', '', ''])
-    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.inactiveReceiveCapacity, NETWORK_STATUSES.INACTIVE), formatBalance(counterSymbolCapacities.inactiveSendCapacity, NETWORK_STATUSES.INACTIVE)])
-    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.inactiveSendCapacity, NETWORK_STATUSES.INACTIVE), formatBalance(counterSymbolCapacities.inactiveReceiveCapacity, NETWORK_STATUSES.INACTIVE)])
+    statusTable.push([`  Buy ${baseSymbol}`, formatBalance(baseSymbolCapacities.inactiveReceiveCapacity, NETWORK_STATUSES.INACTIVE), formatBalance(counterSymbolCapacities.inactiveSendCapacity, NETWORK_STATUSES.INACTIVE)])
+    statusTable.push([`  Sell ${baseSymbol}`, formatBalance(baseSymbolCapacities.inactiveSendCapacity, NETWORK_STATUSES.INACTIVE), formatBalance(counterSymbolCapacities.inactiveReceiveCapacity, NETWORK_STATUSES.INACTIVE)])
 
-    logger.info(` ${market.bold.white}`)
+    logger.info(` Market: ${market.bold.white}`)
     logger.info(statusTable.toString())
-  } catch (e) {
-    logger.error(handleError(e))
-  }
-}
+    logger.info('')
 
-/**
- * Formats the capacities to fixed numbers and colors them if they are greater
- * than 0
- *
- * @function
- * @param {string} balance
- * @param {string} status
- * @return {string} balance formatted with size and color
- */
-function formatBalance (balance, status) {
-  const fixedBalance = Big(balance).toFixed(16)
-  if (Big(balance).gt(0)) {
-    switch (status) {
-      case NETWORK_STATUSES.AVAILABLE:
-        return fixedBalance.green
-      case NETWORK_STATUSES.OUTSTANDING:
-        return fixedBalance.yellow
-      case NETWORK_STATUSES.PENDING:
-        return fixedBalance.yellow
-      case NETWORK_STATUSES.INACTIVE:
-        return fixedBalance.red
-      default:
-        return fixedBalance
+    if (baseSymbolCapacities.status !== 'OK') {
```

Comment:
string for `OK` is a little strange

Reply:
@treygriffith want me to move to an enum?

Reply:
Ive moved the string into an enum